gax/src/main/java/com/google/api/gax/batching/v2/BatchingSettings.java

```diff
@@ -32,17 +32,18 @@
 import com.google.api.core.BetaApi;
 import com.google.auto.value.AutoValue;
 import com.google.common.base.Preconditions;
+import org.threeten.bp.Duration;
 
 /**
  * Represents the batching settings to use for an API method that is capable of batching.
  *
  * <p>Each batching client must define their own set of default values for these thresholds, which
  * would be the safest behavior for their jobs.
  *
- * <p>The default instance of this settings are configured to accept elements until either of the
- * threshold reaches {@link Long#MAX_VALUE}) or an explicit call to {@link Batcher#flush()} is made
- * or the {@link Batcher#close()} is called. Users are expected to configure actual batching
- * thresholds explicitly: the element count or the request bytes count.
+ * <p>The default instance of this settings are configured to accept elements until any of the
```

Comment:
these settings are configured to ...

Reply:
"instance of this" tells about a single instance of `BatchingSettings.java`